examples/integration_test.py

```diff
@@ -806,13 +819,65 @@ def my_on_revoke(consumer, partitions):
     if msgcnt > 0:
         t_spent = time.time() - t_first_msg
         print('%d messages (%.2fMb) consumed in %.3fs: %d msgs/s, %.2f Mb/s' %
-              (msgcnt, bytecnt / (1024*1024), t_spent, msgcnt / t_spent,
-               (bytecnt / t_spent) / (1024*1024)))
+              (msgcnt, bytecnt / (1024 * 1024), t_spent, msgcnt / t_spent,
+               (bytecnt / t_spent) / (1024 * 1024)))
 
     print('closing consumer')
     c.close()
 
 
+def verify_throttle_cb():
+    """ Time how long it takes to produce and delivery X messages """
+    conf = {'bootstrap.servers': bootstrap_servers,
+            'api.version.request': api_version_request,
+            'linger.ms': 500,
+            'client.id': 'throttled_client',
+            'throttle_cb': throttle_cb}
+
+    p = confluent_kafka.Producer(**conf)
```

Comment:
use `..Producer(conf)` instead, looks neater and we're moving away from the `**` notation

Reply:
ack